src/docs/user_ai.md

```diff
@@ -434,6 +435,27 @@ Finally, the setting 'Max Context Lines' allows you to configure the maximum num
 
 This agent assists with writing and executing terminal commands. Based on the user's request, it suggests commands and allows them to be directly pasted and executed in the terminal. It can access the current directory, environment, and recent terminal output to provide context-aware assistance. You can open the terminal assistance agent via Ctrl+I in the terminal view.
 
+### App Tester (Chat Agent)
+
+The App Tester is an AI-driven agent that helps you test browser-based applications using Playwrite (using a Playwrite MCP server). It performs end-to-end (E2E) testing by interacting directly with your running application in a browser, focusing on testing the application through its UI rather than generating unit tests for business logic.
```

Comment:
```suggestion
The App Tester is an AI-driven agent that helps you test browser-based applications using Playwright (using a Playwright MCP server). It performs end-to-end (E2E) testing by interacting directly with your running application in a browser, focusing on testing the application through its UI rather than generating unit tests for business logic.
```